pkg/zson/docs/spec.md

```diff
@@ -1,29 +1,29 @@
-# zson specification
+# ZSON specification
 
-zson is a format for structured data values, ideally suited for streams
+ZSON is a format for structured data values, ideally suited for streams
 of heterogeneously typed records.
-zson is richly typed and thinner than json.
-Like [newline-delimited json (ndjson)](http://ndjson.org/),
-zson represents a sequence of data objects that can be parsed line by line.
+ZSON is richly typed and thinner than JSON.
+Like [newline-delimited JSON (NDJSON)](http://ndjson.org/),
+ZSON represents a sequence of data objects that can be parsed line by line.
 
-zson strikes a balance between the narrowly typed but flexible ndjson format and
+ZSON strikes a balance between the narrowly typed but flexible NDJSON format and
 a more structured approach like
 [Apache Avro](https://avro.apache.org).
-zson is type rich and
+ZSON is type rich and
 embeds all type/schema in the stream, while having a value syntax
-independent of the schema so it is easy and efficient to parse on the fly
+independent of the schema, making it easy and efficient to parse on the fly
 and mix and match streams from different sources with heterogeneous types.
 Like Avro,
-zson embeds schema information in the data stream but zson schema bindings
+ZSON embeds schema information in the data stream but ZSON schema bindings
 are lighter-weight and specified with a simple integer encoding that
-accompanies each each data value.
+accompanies each data value.
 
-The zson design [is also motivated by](./rationale.md)
-and maintains backward compatibility with the original zeek log format.
+The ZSON design [is also motivated by](./rationale.md)
+and maintains backward compatibility with the original [Zeek ASCII TSV log format](https://docs.zeek.org/en/stable/examples/logs/).
```

Comment:
I know internally we often say "Zeek logs" as shorthand for what is effectively "the TSV log format that's output by the Zeek's [ASCII Writer](https://docs.zeek.org/en/stable/frameworks/logging.html#ascii-writer)". This is obviously a mouthful, so shorthand helps. At the same time, considering that Zeek's ASCII writer can also output in JSON (and many customers do this as we know) I figure it doesn't hurt to emphasize for the reader that we're talking specifically about the TSV one here, so "Zeek ASCII TSV" is my attempt at slightly-less-short-shorthand.

Reply:
I think we should avoid TSV because the ASCII log writer has a filter config option called `tsv` that changes the header substantially.

I'd like to suggest "default Zeek log format" or "standard Zeek log format" since it's the format produced by Zeek with the default log filter.

If you want to be specific, I think the only really precise description is "the format produced by Zeek's ASCII log writer when `LogAscii::include_meta` is true, the `meta_prefix` filter config option (when set) or `LogAscii::meta_prefix` (when the filter config option is unset) is `"#"`, the `use_json` filter config option (when set) or `LogAscii::use_json` (when the filter config option is unset) is false, and the `tsv` filter config option is false." Yikes.

Reply:
@nwt: Very helpful. Based on this, I'm prepared to switch the wording to "default Zeek log format".